.github/workflows/build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - name: Check style against standards using prospector
       run: prospector
     - name: Check import order
-      run: isort --recursive --check-only virtual-coach-server
```

Comment:
This would only check the virtual-coach-server folder, but we want to check all python code in the repo.